src/compiler.rs

```diff
@@ -75,14 +86,65 @@ impl Compiler {
               .join(Self::expand_tilde(&relative.cooked)?)
               .lexiclean();
 
-            if import.is_file() {
+            let has_glob = import.as_os_str().as_encoded_bytes().contains(&b'*');
```

Comment:
This is tricky because glob supports more meta-characters than `*`. It supports `?`, `*`, `[`, `!`, and `]`. I suppose we should check for all of them.

Reply:
Can we assume that none of those characters will show up in a legitimate justfile? I think that's safe to assume for `*` but maybe not for characters like `[` and `]`.

Reply:
What if there was a slightly different version of the `import` keyword that signaled that shell globs might be used, or an argument to it? `import glob "my/path/*.just"` or something like that?